src/Api/Invoice.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Billogram\Api;
+
+use Billogram\Exception\Domain\ValidationException;
+use Billogram\Model\Invoice\Invoice as Model;
+use Billogram\Model\Invoice\Invoices;
+
+/**
+ * @author Ibrahim Hizeoui <ibrahimhizeoui@gmail.com>
+ */
+class Invoice extends HttpApi
+{
+    /**
+     * @param array $param
+     *
+     * @return string|array
+     *
+     * @see https://billogram.com/api/documentation#billogram_call_create
+     */
+    public function search(array $param = [])
+    {
+        $param = array_merge(['page' => 1, 'page_size' => 100], $param);
+        $response = $this->httpget('/billogram', $param);
+        $body = $response->getBody()->__toString();
```

Comment:
ok can I change it to here ??
throw new DomainExceptions\NotFoundException($response->getBody()->__toString());